src/components/media-types/container.js

```diff
@@ -5,6 +5,8 @@ import classnames from 'classnames'
 import { HicetnuncContext } from '../../context/HicetnuncContext'
 import styles from './styles.module.scss'
 
+const iPhone = /iPhone|iPod/.test(navigator.userAgent) && !window.MSStream
```

Comment:
dont want to be stubborn, but don't we need iPad here too? https://stackoverflow.com/questions/9038625/detect-if-device-is-ios

Reply:
@pichiste ☝🏻

Reply:
iPad supports fullscreen API strangely, I tested it here! It's just iPhone that is the problem. I know it doesn't make any sense... here's a screenshot from my iPad:

![IMG_0015](https://user-images.githubusercontent.com/111979/113284590-22df6380-92ea-11eb-9883-9a637cbb0919.png)